webapp/src/shared/components/Button/Button.css

```diff
@@ -42,6 +42,4 @@
 
 .button-icon {
   display: flex;
-  position: absolute;
-  left: 1rem;
```

Comment:
praise: Nice detail. Now the icon of the small button is centered :+1:

Reply:
I'd still prefer the large button to keep the style it used to have, otherwise icon and button label almost overlap
Below the necessary change to achieve this.

Reply:
Done!